sunpy/map/header_helper.py

```diff
@@ -7,10 +7,12 @@
 from sunpy import log
 from sunpy.coordinates import frames, sun
 from sunpy.util import MetaDict
+from sunpy.util.decorators import deprecated
 
 __all__ = ['meta_keywords', 'make_fitswcs_header', 'get_observer_meta', 'make_heliographic_header']
 
 
+@deprecated(since="5.0", message="Unused and will be removed in 6.0")
```

Comment:
The unit test for this function needs to use `pytest.warns` to catch and check for the deprecation warning.